src/test/java/com/cdancy/jenkins/rest/features/JobsApiLiveTest.java

```diff
@@ -39,18 +41,21 @@
 public class JobsApiLiveTest extends BaseJenkinsApiLiveTest {
 
     private Integer queueId;
+    private Integer queueIdForAnotherJob;
     private Integer buildNumber;
+    private static final String FOLDER_PLUGIN_NAME = "cloudbees-folder";
+    private static final String FOLDER_PLUGIN_VERSION = "6.4";
```

Comment:
What about using `latest` here instead of an actual version? That's very non-ci but I'd rather chase the latest and greatest version instead of something which may become out of date over time.